CMakeLists.txt

```diff
@@ -774,7 +785,8 @@ IF(USE_HDF5)
       MESSAGE(FATAL_ERROR "netCDF requires at least HDF5 ${HDF5_VERSION_REQUIRED}. Found ${HDF5_VERSION}.")
     ENDIF()
 
-
+    SET(HDF5_HAS_SWMR ${ENABLE_HDF5_SWMR})
```

Comment:
This enables `SWMR` when finding through `find_package`, you need to duplicate in the other `if` branch to also enable when `HDF5` paths are set explicitely

Reply:
Thanks, good catch!